environments/test/service-core-compute-perftest-internal.yml

```
@@ -39,4 +39,8 @@ A:
     record:
     - 10.61.32.121
     ttl: 300
+  - name: idam-keycloak-perftest.service.core-compute-perftest.internal
```

that will get you a huge doubled dns record
```suggestion
  - name: idam-keycloak-perftest
```